celery/app/task.py

```diff
@@ -764,7 +764,12 @@ def apply(self, args=None, kwargs=None,
             'callbacks': maybe_list(link),
             'errbacks': maybe_list(link_error),
             'headers': headers,
-            'delivery_info': {'is_eager': True},
+            'delivery_info': {
+                'is_eager': True,
+                'exchange': options.get('exchange'),
```

Comment:
what do these changes do? should we mention the changes in docs?

Reply:
They just make these options available inside the task when running a task eagerly:

```
@shared_task(bind=True)
def my_task(self):
     print(self.delivery_info.get('priority'))
     
my_task.apply_async(priority=2) # prints 2 in v5.0.0
my_task.apply(priority=2) # prints None v5.0.0
```

Reply:
I agree.